### ISSUE TITLE
Resource count not updating on latest Tidy5e character sheet

### ISSUE THREAD

**Issue by flamewave000** [ardittristan/5eSheet-resourcesPlus#10]
Tidy5e shows all of the 20 resources, instead of just the 5 I've set in the character sheet settings.

![image](https://user-images.githubusercontent.com/3588046/103709743-490e9500-4f81-11eb-8c03-cd91be55e2bb.png)

**Comment by Ghostdesu** [comment]
As a temporary fix you can set global resource limit to the minimum you need, it'll override broken sheeet settings
![image](https://user-images.githubusercontent.com/31375910/103713566-55b4dc80-4fc5-11eb-80a1-babc0a4258d2.png)
![image](https://user-images.githubusercontent.com/31375910/103713569-56e60980-4fc5-11eb-848f-53b52b1dbf5d.png)

**Comment by ardittristan** [comment]
hm, did you try changing one of the resource values? generally it works after then. And it also might take a few opens and closes of the sheet to initialize the values correctly. This is because it's basically overwriting a core foundry function.

**Comment by flamewave000** [comment]
I've set values, and open/closed these sheets several times. The issue persists. It was working a while ago, but I updated my server to 0.7.9 and updated the Tidy5e sheet and Resources+ when I first saw the issue. I also tried reinstalled both modules with no luck. I know the Tidy5e sheet underwent a pretty heavy face-lift, so I'm thinking it may just be a conflicting issue between the two modules, as the Resources+ works perfect with the other sheets I've tried it with.
